.github/workflows/tests.yml

```diff
@@ -31,5 +31,5 @@ jobs:
             Makefile
       - name: Run Go Tests
         run: |
-          make test-group-${{ matrix.part }} NUM_SPLIT=6
+          make test-group-${{ matrix.part }} NUM_SPLIT=20
```

Comment:
what is the "ideal" split size?

Reply:
Seemingly 20.  We could go higher but I don't think there's really any need, once we go up to 20 time used is about half tests and half setup.